backend/database-sqlite.js

```diff
@@ -2,89 +2,63 @@
 
 import { mkdir } from 'node:fs/promises'
 import { join, resolve } from 'node:path'
-import sqlite3 from 'sqlite3'
+import Database from 'better-sqlite3'
+import DatabaseBackend from './DatabaseBackend.js'
+import type { IDatabaseBackend } from './DatabaseBackend.js'
 
-let db: any = null
-let readStatement: any = null
-let writeStatement: any = null
-let deleteStatement: any = null
+export default class SqliteBackend extends DatabaseBackend implements IDatabaseBackend {
+  dataFolder: string = ''
+  db: any = null
+  filename: string = ''
+  readStatement: Object = null
+  writeStatement: Object = null
+  deleteStatement: Object = null
 
-const run = (sql, args) => {
-  return new Promise((resolve, reject) => {
-    db.run(sql, args, (err) => {
-      if (err) {
-        reject(err)
-      } else {
-        resolve()
-      }
-    })
-  })
-}
+  constructor (options: Object = {}) {
+    super()
+    const { dirname, filename } = options
+    this.dataFolder = resolve(dirname)
+    this.filename = filename
+  }
+
+  run (sql: string) {
```

Comment:
Why not just use `this.db.exec(sql)` (i.e., why `prepare` + `run`?)

Reply:
IIUC, the [docs](https://github.com/WiseLibs/better-sqlite3/blob/master/docs/api.md#execstring---this) recommend to avoid using `exec` if possible.